readme.txt

```diff
@@ -71,7 +71,7 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
 
 == Changelog ==
 
-= 1.3.0-beta.1 2020-06-15 =
+= 1.3.0 2020-07-08 =

```

Comment:
Here are some that were included in the RC's not added to this list.

- Dev: Check that the possibly_add_note function exists before calling it #4680
- Dev: Remove unnecessary rest API init action. #4691
- Dev: Don't include sourcemaps or unminified JS for "core" builds. #4642
- Tweak: Remove duplicate/redundant inbox note after first order received. #4659
- Tweak: Fix the embed page CSS so the top content sits better #4622
- Fix: Fix WCPay sometimes not appearing on the task list #4647

Reply:
Thanks Paul, i'll update those and merge them in. I think they might be missing from the WPORG changelog too since the list in this branch came from `release/1.3.0`